plugin/someteam.example.com/v1/starlarkmixer/go.mod

```diff
@@ -0,0 +1,7 @@
+module sigs.k8s.io/kustomize/plugin/someteam.example.com/v1/starlark_mixer
+
+go 1.15
+
+require sigs.k8s.io/kustomize/api v0.7.2
+
+replace sigs.k8s.io/kustomize/api => ../../../../api
```

Comment:
i believe the presubmit check is failing because it is using an older version of the kyaml module

try adding to this file
`replace sigs.k8s.io/kustomize/kyaml => ../../../../kyaml`

You can run `make prow-presubmit-check` to see if that fixes the problem

Reply:
That seems to have done the trick. Thanks for the tip!